.husky/_/husky.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
```

Comment:
Is this for the pre-commit hook?

Reply:
yes, just a husky script for the pre-commit hook 😄